arabic/unpacked/common-identifiers.js

```diff
@@ -0,0 +1,54 @@
+MathJax.Hub.Config({
+  // Math variables and functions
+  Arabic: {
+    identifiersMap: {
+      // Sets operations, and other stuff
+      'A': 'أ',
```

Comment:
- [ ] Sure, I can do that. I might do it as a postprocessing step along with the other [gulp steps that I have](https://github.com/Edraak/arabic-mathjax-dev/blob/b71ddcefdd27ba9e8507d29d507d97cd0b924572/gulpfile.js).

It is really handy for me to see the original Arabic text instead of unicode points. In production this is not a requirement, so I can post process this.